src/main/java/helper/SuccessResponseGenerator.java

```diff
@@ -78,10 +80,37 @@
             responseBody.put("SentByCurrentUser", false);
         }
 
+        responseBody.put("SentTs", message.getSentTs());
+        responseBody.put("ReceivedTs", message.getReceivedTs());
+        responseBody.put("CreationTs", message.getCreationTs());
         responseBody.put("TextContent", message.getTextContent());
+
+        return responseBody;
+    }
+
+    /**
+     * Renders the given parameters in a Map to return a successful HTTP response 
+     * for all client requests to the GetMessage API for messages with attachments.
+     */
+    public static Map<String, Object> getSuccessResponseForGetMessage(Message message, Attachment attachment, long userId) {
```

Comment:
The code for this function is same as the other one plus some stuff.
Why not call it first and fill the other things afterwards instead of repeating the code?

Reply:
Done